### ISSUE TITLE
Prevent touch screen input while magnifier is running and document limitation

### ISSUE THREAD

**Issue by seanbudd** [nvaccess/nvda#20038]
### Is your feature request related to a problem? Please describe.

Touch screen doesn't work with NVDA magnifier currently (see #19491).
This causes incorrect mappings for touch screen input.

### Describe the solution you'd like

Disable touch screen input while magnifier is running.
Document this limitation in the user guide.

We can't disable touch input on portable copies, so an alternative solution is needed here

**Comment by Boumtchack** [comment]
I think we can close this issue as we resolved it in https://github.com/nvaccess/nvda/pull/20357

**Comment by SaschaCowley** [comment]
@Boumtchack this will close automatically once it hits master.
